src/main/java/uk/gov/ch/developer/docs/controller/developer/SignInController.java

```diff
@@ -65,7 +65,8 @@ void redirectForAuth(final Session session, final HttpServletRequest request,
     }
 
     /**
-     * Constructs the authorisation URI with additional force and hint
+     * Constructs the authorisation URI with additional force and hint Not used anywhere, except in
+     * a test.
```

Comment:
This can be refactored out if not used anywhere, the original plan was to have this for future proofing the library. I think this was related to hijack protection (I can't remember at the moment).

Reply:
Although that is out of scope for this ticket

Reply:
Incorrect capitalisation between hint and Not. Try a full stop